src/test/java/raisetech/student/management/controller/StudentsControllerTest.java

```diff
@@ -76,14 +78,59 @@ class StudentsControllerTest {
   }
 
   @Test
-  void 受講生検索ができて空で帰ってくること() throws Exception {
+  void 受講生のID検索ができて空で帰ってくること() throws Exception {
     String id = "1";
-    mockmvc.perform(get("/students/{id}", id))
+    mockmvc.perform(get("/studentsById/{id}", id))
         .andExpect(status().isOk())
      .andExpect(content().string(""));
 
     verify(service, times(1)).findStudentById(id);
   }
+  @Test
+  void 受講生の名前検索ができて空で帰ってくること() throws Exception {
```

Comment:
以下、typoですね。(他のテストケースも...)
```suggestion
  void 受講生の名前検索ができて空で返ってくること() throws Exception {
```